geomesa-kafka/geomesa-kafka-confluent/src/test/scala/org/locationtech/geomesa/kafka/confluent/ConfluentKafkaDataStoreFactoryTest.scala

```diff
@@ -0,0 +1,116 @@
+package org.locationtech.geomesa.kafka.confluent
```

Comment:
build is failing due to a missing license header in this file. you should be able to run `mvn license:format` to add it.